theano/tensor/signal/pool.py

```diff
@@ -113,30 +114,103 @@ def pool_2d(input, ds, ignore_border=None, st=None, padding=(0, 0),
     return tensor.reshape(output, outshp, ndim=input.ndim)
 
 
-class Pool(OpenMPOp):
+def pool_3d(input, ds, ignore_border=None, st=None, padding=(0, 0, 0),
```

Comment:
I don't have clear enough overview of the consequences, but are you really sure that is a good idea? That would give two almost identical functions, the only difference 2D/3D, with a very subtle difference in one of the default parameters.